### PR TITLE
Fix COPY with Depth 0

### PR DESCRIPTION
**Pull request by PVince81**
Honor the Depth header in Webdav COPY operation.
A Depth 0 means only duplicate the collection and not its contents.
A Depth infinity means also copy its contents.

@butonic

### CODE REVIEW

**Comment by PVince81** [comment]
note: copy with depth infinity is still broken, raised here: https://github.com/owncloud/ocis-reva/issues/173

File: internal/http/services/owncloud/ocdav/copy.go [hunk 1]
```diff
@@ -153,7 +163,7 @@ func (s *svc) handleCopy(w http.ResponseWriter, r *http.Request, ns string) {
 		// TODO what if intermediate is a file?
 	}
 
-	err = descend(ctx, client, srcStatRes.Info, dst)
+	err = descend(ctx, client, srcStatRes.Info, dst, depth == "infinity")
```
Comment (labkode) [inline internal/http/services/owncloud/ocdav/copy.go:166]: Shouldn't we descend also in depth > 1?

Comment (PVince81) [reply internal/http/services/owncloud/ocdav/copy.go:166]: the spec doesn't allow any other values than "0" and "infinity": https://tools.ietf.org/html/rfc4918#section-9.8.3

Comment (labkode) [reply internal/http/services/owncloud/ocdav/copy.go:166]: @PVince81 Depth 1 is used by the sync clients

**Pull request merged**